ArkhamOverlay/Utils/OutlinedTextBlock.cs

```diff
@@ -223,6 +223,7 @@ public class OutlinedTextBlock : FrameworkElement {
             outlinedTextBlock.InvalidateVisual();
         }
 
+        [Obsolete]
```

Comment:
What is the purpose of this?

Reply:
I dunno, I copied this off the interenet- lol